review.py

```diff
@@ -0,0 +1,164 @@
+import random
+import time
+
+
+class Player:
+    def __init__(self, name, hp, mp, phys_attack, accuracy, defense):
+        self.name = name
+        self.level = 1
+        self.dream_level = 1
+        self.hp = hp
+        self.mp = mp
+        self.phys_attack = phys_attack
+        self.accuracy = accuracy
+        self.defense = defense
+        self.inventory = {}
+        self.equipped = {"head": None, "neck": None, "chest": None,
+                         "arm_r": None, "arm_l": None, "hands": None,
+                         "hand_r": None, "hand_l": None, "waist": None,
+                         "legs": None, "feet": None}
+
+    def adjust_stats(self, item, remove=False):
+        is_weapon = isinstance(item, Weapon)
+        if remove is True:
+            if is_weapon:
+                self.phys_attack -= item.attack_boost
+                self.accuracy -= item.accuracy_boost
+            else:
+                self.hp -= item.hp_boost
+                self.mp -= item.mp_boost
+                self.defense -= item.defense_boost
+        else:
+            if is_weapon:
+                self.phys_attack += item.attack_boost
+                self.accuracy += item.accuracy_boost
+            else:
+                self.hp += item.hp_boost
+                self.mp += item.mp_boost
+                self.defense += item.defense_boost
+
+    def equip_item(self, item):
```

Comment:
This function has a few different types of thing going on:

1. Check to see if this `item` is a legit thing we can equip
2. Take input from the player
3. Modify the equipped items

I think at least `2` might be easier to think about if you split it out into a separate method.